core/src/main/java/org/jahia/modules/modulemanager/flow/ModuleManagementFlowHandler.java

```diff
@@ -329,7 +356,7 @@ private ModuleInstallationResult installModule(File file, MessageContext context
             if (symbolicName == null) {
                 symbolicName = manifest.getMainAttributes().getValue(Constants.ATTR_NAME_ROOT_FOLDER);
             }
-            String version = manifest.getMainAttributes().getValue(Constants.ATTR_NAME_IMPL_VERSION);
+            String version = StringUtils.defaultIfBlank(manifest.getMainAttributes().getValue(Constants.ATTR_NAME_BUNDLE_VERSION), manifest.getMainAttributes().getValue(Constants.ATTR_NAME_IMPL_VERSION));
```

Comment:
I don't know in which case they are different, but taking Bundle-Version instead of Implementation-Version might have an impact when deploying standard modules - is it really needed to change that ?

Reply:
We need the version in order to move forward with the block below and when installing a tgz module the "getValue(Constants.ATTR_NAME_IMPL_VERSION)" is empty. I tried to install standard jar modules and it still working. 

```
                ModuleVersion moduleVersion = new ModuleVersion(version);
                Set<ModuleVersion> allVersions = templatePackageRegistry.getAvailableVersionsForModule(symbolicName);
                if (!forceUpdate) {
                    if (!moduleVersion.isSnapshot()) {
                        if (allVersions.contains(moduleVersion)) {
                            context.addMessage(new MessageBuilder().source("moduleExists")
                                    .code("serverSettings.manageModules.install.moduleExists")
                                    .args(symbolicName, version)
                                    .build());
                            return null;
                        }
                    }
                }
```

Reply:
Actually, I believe it should be the opposite. 

Instead "ATTR_NAME_BUNDLE_VERSION" be the first checked if is blank we should try ATTR_NAME_IMPL_VERSION first

`String version = StringUtils.defaultIfBlank(manifest.getMainAttributes().getValue(Constants.ATTR_NAME_IMPL_VERSION), manifest.getMainAttributes().getValue(Constants.ATTR_NAME_BUNDLE_VERSION));`